integration_tests/pages/apply/taskListPage.ts

```diff
@@ -32,7 +32,7 @@ export default class TaskListPage extends Page {
   }
 
   shouldShowTaskStatus = (task: string, status: string): void => {
-    cy.get(`#${task}-status`).should('contain', status)
+    cy.get(`#${task}-status`).debug().should('contain', status)
```

Comment:
is this intentional?

Reply:
Not sure how this got here, looks like some funkiness with git history, as this was fixed in an earlier PR. I'll drop the commit.